### PR TITLE
Add option to skip Xbox boot animation.

### PR DESCRIPTION
**Pull request by mborgerson**

### CODE REVIEW

File: vl.c [hunk 1]
```diff
@@ -440,6 +440,10 @@ static QemuOptsList qemu_machine_opts = {
             .name = "mediaboard_filesystem",
             .type = QEMU_OPT_STRING,
             .help = "Chihiro mediaboard filesystem file",
+        },{
+            .name = "skipanim",
```
Comment (espes) [inline vl.c]: nit: call this short_animation and put xbox in the help string

File: hw/xbox/smbus_xbox_smc.c [hunk 2]
```diff
@@ -137,9 +138,16 @@ static uint8_t smc_read_data(SMBusDevice *dev, uint8_t cmd, int n)
             /* pretend to have a composite av pack plugged in */
             return SMC_REG_AVPACK_COMPOSITE;
 
-        case SMC_REG_SCRATCH:
+        case SMC_REG_SCRATCH: {
+            /* Skip boot animation if the "skipanim" flag was set. */
+            QemuOpts *opts = qemu_opts_find(qemu_find_opts("machine"), NULL);
```
Comment (espes) [inline hw/xbox/smbus_xbox_smc.c]: you should query this in smbus_smc_init

Comment (mborgerson) [inline hw/xbox/smbus_xbox_smc.c]: Pardon my sloppy hack.

**Comment by espes** [comment]
Thanks!

**Pull request merged**